sass/components/_card.scss

```diff
@@ -1,13 +1,31 @@
 .card {
+    @include flex(column, left, left);
     position: relative;
-    max-width: 40rem;
+    max-width: 37rem;
     padding: 2rem;
-    margin-bottom: 35rem;
+    box-shadow: $bs-bg-box;
+    border-radius: 2.6rem;
+    box-shadow: $bs-bg-box2, $bs-bg-box;
```

Comment:
powtarzający się `box-shadow`
stworzyłbym ten efekt jako komponent w pliku `decorations` jako osobną klase,
np. sposób prosto z figmy wykonać na pseudoelemencie `before`:
`background-color`: #59B1E6;
`opacity`: 0.8;
`filter`: blur(150px);
`position`: absolute;
`top`: -59px;
`right`: -44px;

umożliwi to w przyszłości bezproblemowe użycie efektu na dowolnym elemencie przy łatwiejszym pozycjonowaniu (szybciej jest manipulować położeniem za pomocą `top`, `right` etc.  zamiast zmieniać lub tworzyć nowy `box-shadow`)

Reply:
done